tests/datasets/test_spacenet.py

```diff
@@ -142,6 +149,75 @@ def test_plot(self, dataset: SpaceNet2) -> None:
         plt.close()
 
 
+class TestSpaceNet3:
+    @pytest.fixture(
+        params=itertools.product(["PAN", "MS", "PS-MS", "PS-RGB"], [False, True])
```

Comment:
For other datasets we've been using `zip` or `itertools.zip_longest` so that we get all coverage with the fewest number of tests. `itertools.product` is fine as long as these tests are very fast.

Reply:
Nice! This reduced the number of tests from 8 to 2.